packages/govuk-frontend/src/govuk/helpers/media-queries.unit.test.js

```diff
@@ -100,27 +116,34 @@ describe('@mixin govuk-media-query', () => {
     })
   })
 
-  it('allows you to target combined min-width and max-width using numeric values', async () => {
-    const sass = `
+  it.each([
+    ['20em', '40em', '20em', '40em'],
+    ['20px', '40px', '1.25em', '2.5em'],
+    ['20', '40px', '1.25em', '2.5em']
```

Comment:
Based on the prior tests it seems like these are meant to be unquoted numbers. Is that the case? 

```suggestion
    [20, 40, '1.25em', '2.5em']
```

Reply:
Ah yeah, was trying to test mixed inputs. I'll use an unquoted number and the px unit.